Tella/Data/VaultManager/VaultFilesManagerExtension.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  VaultFilesManagerExtension.swift
+//  Tella
+//
+//  Created by Dhekra Rouatbi on 13/6/2024.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import MobileCoreServices
+import Photos
+
+extension VaultFilesManager {
+    
+    func updateURL(importedFile:inout ImportedFile) async {
+        
+        do {
+            
+            var url : URL?
+            guard let asset = importedFile.asset else {
+                return
+            }
+            
+            let mediaType = asset.mediaType
+            
+            if (importedFile.urlFile != nil && mediaType == .image) {
+                return // Image data already saved into the temp fileURL
+            }
+            
+            switch asset.mediaType {
+            case .image:
+                url = try await getImageUrlFromAsset(importedFile:importedFile)
```

Comment:
Instead of creating a url variable let's just assign it directly to `importedFile.urlFile`. it's less code :)